src/constants.ts

```diff
@@ -53,6 +54,10 @@ export enum MESSAGE_TYPE {
   VALIDATE_WALLET_NAME,
 }
 
+export enum RESPONSE_TYPE {
+  LOADING,
```

Comment:
can we call this `RESTORING_SESSION` or something like that?

Reply:
mm I was thinking of calling it a general loading enum so that we could reuse it if we need to wait for a different background process in the future. What do you think? It is minor, so I don't mind change it to  RESTORING_SESSION  if you prefer that

Reply:
I think it makes sense to give the return of the response, which is checking if it is restoring session. And changing to the loading screen is a byproduct of getting a restoring session response.